SourceX/DiabloUI/dialogs.cpp

```diff
@@ -185,9 +168,14 @@ void DialogLoop(UiItem *items, std::size_t num_items, UiItem *render_behind, std
 				exit(0);
 			}
 		}
-		UiRenderItems(render_behind, render_behind_size);
+
+		if (render_behind_size == 0) {
+			SDL_FillRect(pal_surface, nullptr, 0);
+		} else {
+			UiRenderItems(render_behind, render_behind_size);
+		}
 		UiRenderItems(items, num_items);
-		DrawMouse();
+        DrawMouse();

```

Comment:
nit: this line uses spaces for indentation but everywhere else it's tabs

Reply:
Fixed, on a different system then usual so didn't have my editor setup correctly 👍